src/main/java/org/alliancegenome/curation_api/services/helpers/DtoConverterHelper.java

```diff
@@ -19,4 +20,15 @@
                     return syn;
                 }).collect(toList());
     }
+    
+    public static List<Synonym> getSynonyms(MoleculeDTO molecule) {
```

Comment:
This method is not needed if you are going to use the `List<String>` for synonyms?

Reply:
removed